### PR TITLE
docs: update readme and package.json

### PR DESCRIPTION
**Pull request by KFlash**

### CODE REVIEW

**Review by 3cp — APPROVED** [review]

**Comment by 3cp** [comment]
ci said package-lock is not in sync, pls do `npm i` and commit lock file.

File: package.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
   },
   "types": "dist/src/meriyah.d.ts",
   "typings": "dist/src/meriyah.d.ts",
-  "author": "KFlash",
```
Comment (3cp) [inline package.json:16]: Removed author but didn't add to authors?

**Comment by 3cp** [comment]
Maybe just add package-lock to gitignore?

File: package.json (hunk 1 above)
Comment (KFlash) [reply package.json:16]: > ci said package-lock is not in sync, pls do `npm i` and commit lock file.

I can't do it from a cp. Can you fix it?

> Removed author but didn't add to authors?

Didn't figure out how to write it in authors. It's only a nick that one. Can you take a stab at it?

**Comment by KFlash** [comment]
> Maybe just add package-lock to gitignore?

Is it safe?

**Comment by 3cp** [comment]
The worst result is just some 3rd party dep breaking our CI, hardly any effect on end users.

Removing lock file may help us to detect 3rd party breaking change earlier.

Lock file makes more sense in app project which you want to build and deploy with confidence.

**Comment by KFlash** [comment]
You can fix it? BTW. Circleci got a problem?

**Comment by 3cp** [comment]
package.json has author and contributors, no authors. https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors

**Comment by KFlash** [comment]
I think this can be merged ?

**Pull request merged**